src/ray/protobuf/ray_syncer.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+option cc_enable_arenas = true;
+package ray.rpc.syncer;
+
+enum RayComponentId {
+  RESOURCE_MANAGER = 0;
+  SCHEDULER = 1;
+}
+
+enum RaySyncMessageType {
```

Comment:
might worth define the behavior a bit clear:
```
For Broadcast type, the syncer will broadcast the message as it is.
For Aggregate type, the syncer will merge messages with the same component_id.
```

Think it another again, do we need to pass this in per syncer message? it could be part of syncer's implementation that this syncing RPC doesn't need to be aware of.